cli/cmd/inject_test.go

```diff
@@ -38,6 +38,11 @@ func testUninjectAndInject(t *testing.T, tc injectYAML) {
 	if expectedOutput != actualOutput {
 		t.Errorf("Result mismatch.\nExpected: %s\nActual: %s", expectedOutput, actualOutput)
 	}
+	if actualOutput != expectedOutput && updateFixtures {
```

Comment:
Thanks for adding the inject fixtures. There are multiple tests in this file, and for this flag to be effective, we should update all of them. It might be worth refactoring the fixture logic a bit so that we can do the rewrites in one place.